### PR TITLE
Update after AMQ Streams 2.7.0 release

### PR DESCRIPTION
**Pull request by jankalinic**

### CODE REVIEW

File: ci-operator/config/amq-streams-qe/strimzi-kafka-operator/amq-streams-qe-strimzi-kafka-operator-main__amq-streams-ocp4.14-lp-rosa-classic.yaml
Comment (im-konge) [inline ci-operator/config/amq-streams-qe/strimzi-kafka-operator/amq-streams-qe-strimzi-kafka-operator-main__amq-streams-ocp4.14-lp-rosa-classic.yaml:61]:
```suggestion
      OLM_OPERATOR_VERSION: 2.7.0-1
```

File: ci-operator/config/amq-streams-qe/strimzi-kafka-operator/amq-streams-qe-strimzi-kafka-operator-main__amq-streams-ocp4.16-lp-interop.yaml
Comment (im-konge) [inline ci-operator/config/amq-streams-qe/strimzi-kafka-operator/amq-streams-qe-strimzi-kafka-operator-main__amq-streams-ocp4.16-lp-interop.yaml:52]:
```suggestion
      OLM_OPERATOR_VERSION: 2.7.0-1
```

**Review by im-konge — COMMENTED** [review]
There is a CVE respin for the operator, so you will need to bump it.

**Review by im-konge — APPROVED** [review]

**Comment by jankalinic** [comment]
/pj-rehearse periodic-ci-amq-streams-qe-strimzi-kafka-operator-main-amq-streams-ocp4.14-lp-rosa-classic-amq-streams-2-7-rosa-aws periodic-ci-amq-streams-qe-strimzi-kafka-operator-main-amq-streams-ocp4.16-lp-interop-amq-streams-2-7

**Comment by jankalinic** [comment]
Blocked until https://github.com/openshift/release/pull/52818 is merged.

**Comment by jankalinic** [comment]
/pj-rehearse periodic-ci-amq-streams-qe-strimzi-kafka-operator-main-amq-streams-ocp4.16-lp-interop-amq-streams-2-7 periodic-ci-amq-streams-qe-strimzi-kafka-operator-main-amq-streams-ocp4.14-lp-rosa-classic-amq-streams-2-7-rosa-aws

**Comment by jankalinic** [comment]
/pj-rehearse periodic-ci-amq-streams-qe-strimzi-kafka-operator-main-amq-streams-ocp4.16-lp-interop-amq-streams-2-7 periodic-ci-amq-streams-qe-strimzi-kafka-operator-main-amq-streams-ocp4.14-lp-rosa-classic-amq-streams-2-7-rosa-aws

**Comment by jankalinic** [comment]
/pj-rehearse periodic-ci-amq-streams-qe-strimzi-kafka-operator-main-amq-streams-ocp4.16-lp-interop-amq-streams-2-7 periodic-ci-amq-streams-qe-strimzi-kafka-operator-main-amq-streams-ocp4.14-lp-rosa-classic-amq-streams-2-7-rosa-aws

**Comment by jankalinic** [comment]
Can we please request merge on this PR? I think we have satisfied all the checks. Thank you. @jmguzik

**Comment by im-konge** [comment]
@jmguzik could you please have a look on this PR?

**Comment by jmguzik** [comment]
I am not sure why I am assigned here. TP is not a direct owner of this dir and teams manage their dirs independently.

**Comment by im-konge** [comment]
Well when I approve it, the PR is still not approved. So I'm not sure who we should ping or how it should work, but for the previous PRs it was always someone who manages this repository or something like that.

**Comment by jmguzik** [comment]
You should ping OWNERS: https://github.com/openshift/release/blob/master/ci-operator/config/amq-streams-qe/strimzi-kafka-operator/OWNERS approvers are mentioned there.

**Comment by im-konge** [comment]
I'm not saying that it's you, we are just following the instructions from the bot:
```
[APPROVALNOTIFIER] This PR is NOT APPROVED

This pull-request has been approved by: https://github.com/openshift/release/pull/52660#issuecomment-2157799197, https://github.com/openshift/release/pull/52660#issuecomment-2147537085, https://github.com/openshift/release/pull/52660#
Once this PR has been reviewed and has the lgtm label, please ask for approval from [jmguzik](https://github.com/jmguzik). For more information see [the Kubernetes Code Review Process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process).
```

That's why you are assigned here. We don't know about who is responsible where.

**Review by dfrazzette — APPROVED** [review]

**Pull request merged**
